### ISSUE TITLE
rOpenGov/estc??

### ISSUE THREAD

**Issue by bhughesshelton** [COMHIS/bibliographica#15]
Hi,
I was just wondering what happened to your repo over at http://github.com/rOpenGov/estc. I'm doing some computational bibliography and found your article, "A Quantitative Study of History in the English Short-Title Catalogue (ESTC), 1470-1800." I had a look at the source code a few weeks ago, but the repo seems to be gone now. Is there any way you can send me the src or allow me to fork the repo? Apologies if this isn't the right venue for this kind of question.

**Comment by antagomir** [comment]
Hi ! Yes the repository was permanently moved to http://github.com/COMHIS/estc very recently and we are still updating all cross-linkings. Apoloiogies for the hassle. Let us know if we can provide support,

**Comment by antagomir** [comment]
However, note that this code and analyses relies on data that is not public. We got the data via confidential collaboration agreement. Therefore, the estc repository itself has mostly information value but does not allow reproducing the analysis in the paper, unless you have your own copy of the data.

**Comment by bhughesshelton** [comment]
Thanks,
I have the complete ESTC already--UC Riverside was kind enough to provide
me with it. I appreciate you sending me the new link to the repo. I've
written some pretty kludgey Perl scripts (
https://github.com/bhughesshelton/ESTC/blob/master/ESTCmeta.pl) to handle
the data, but am looking forward to seeing what your code can do. Any
suggestions for normalizing historical name data?

Cheers,
Barry

**Comment by antagomir** [comment]
Thanks for your interest! We are now reorganizing the code and the complete workflow is at the moment not replicable for various technical reasons. The aim is to really get this set up for the complete data cleaning process and we are working on it.

If you are interested in specific fields, I can see what we could do. Do you refer to historical person names, place names, or something else ?

**Comment by drmhill** [comment]
I'm actually working on this aspect of the ESTC right now. Out of curiosity, what is your goal of normalizing spelling? Having unique identifiers for each author?

**Comment by antagomir** [comment]
Great to hear! Might be useful to compare the matchings up to 1641 at least as our procedure is largely automated whereas yours seems to be manual. This would provide some quality control. It would also be helpful to check through our lists to spot possible mistakes. This is now ongoing and presumably ready rather soon.

**Comment by bhughesshelton** [comment]
Well, the names of authors are already (mostly) standardized in the
ESTC--what I've done is extract and standardize the names of the other
people associated with each text: printers, publishers and booksellers. And
yes, I then assigned everyone a UUID, finally tying that number back to the
UUID for each text, i.e. the ESTC number.

**Comment by bhughesshelton** [comment]
Right, developing an automated process for historical name disambiguation
would be almost impossible (and you'd end up with loads of mistakes). What
I've done is run some fancy pattern matching routines across the MARC
records to extract the information I was interested in, and drive into a
relational db. Then I was able to open an JDBC connection between that db
and a spreadsheet where I could sort the names and copy/paste the most
frequent spelling over the less frequent ones.

**Comment by antagomir** [comment]
Yes that's the key & what we do as well: automate as much as possible, and do the rest by hand. But some degree of automation is crucial here.
